Cargo.toml

```diff
@@ -6,9 +6,9 @@ members = [
 	"libs/core",
```

Comment:
core's path will also be `core/core`. Additionally I see no changes to makefiles which I know will cause problems

Reply:
releaser will also need to be updated